multipath/multipath.conf.5.in

```diff
@@ -187,7 +187,7 @@ The default is: \fBno\fR
 .
 .TP
 .B multipath_dir
-(Deprecated) This option is not supported any more, and the value is ignored.
+(Deprecated) This option is not supported anymore, and the value is ignored.
```

Comment:
It's controversial whether "any more" is wrong, even with the meaning "no longer". The [Cambridge dictionary](https://dictionary.cambridge.org/de/grammatik/britisch-grammatik/any-more-or-anymore#google_vignette) says both spellings are possible.

Whatever.